### PR TITLE
feat: i18n-dutch support

### PR DESCRIPTION
**Pull request by Asmit2952**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR adds Dutch language support for Novu.
- **Why was this change needed?** (You can also link to an open issue here)
This PR fixes: #1568

### CODE REVIEW

**Comment by davidsoderberg** [comment]
Hi @Asmit2952, I see on your Github profile that you are living in India 😄 Have you studied Dutch somewhere?

**Comment by Asmit2952** [comment]
> Hi @Asmit2952, I see on your Github profile that you are living in India 😄 Have you studied Dutch somewhere?

Took some extra classes when I was in my high school 😁

**Comment by Asmit2952** [comment]
I know a bit of Spanish and German as well 😁

**Review by davidsoderberg — APPROVED** [review]

File: packages/notification-center/src/i18n/languages/nl.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const NL: ITranslationEntry = {
+  translations: {
+    notifications: 'Meldingen',
```
[the remark this answers is not in the document]
Comment (Asmit2952) [reply packages/notification-center/src/i18n/languages/nl.ts:5]: Ohh yeah, my translation translates to announcement. Will fix that.

File: docs/docs/notification-center/react-components.md [hunk 2]
```diff
@@ -173,6 +173,7 @@ The `i18n` prop can accept 2 different types of values
           <li><code>ar</code> (Arabic)</li>
           <li><code>de</code> (German)</li>
           <li><code>en</code> (English)</li>
+          <li><code>nl</code> (Dutch)</li>
```
[the remark this answers is not in the document]
Comment (Asmit2952) [reply docs/docs/notification-center/react-components.md:176]: Yupp

**Comment by Asmit2952** [comment]
Resolved all conflicts @p-fernandez

File: packages/notification-center/src/i18n/lang.ts [hunk 3]
```diff
@@ -171,6 +173,9 @@ export type I18NLanguage =
   | 'as'
   | 'no'
   | 'te'
+  | 'da'
+  | 'vi'
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/lang.ts:177]: Remove these 2 please, as they are repeated in the Union.

**Review by p-fernandez — CHANGES_REQUESTED** [review]

**Review by scopsy — APPROVED** [review]
Thank you so much for your contribution! Really appreciate it 🙏

**Pull request merged**